postgresql/init.sls

```diff
@@ -33,7 +33,9 @@ postgresql:
 
 {% if pg_version|float < 9.3 %}
```

Comment:
Some people have 9.3-based servers out there which don't have any configure_utf-8.sh script on board. Seeing one might make them freak out :) Since the next step of the deprecation plan would be to remove the script entirely, I figured those servers didn't need this no-op script.